Packs/Mimecast/Integrations/MimecastV2/MimecastV2.py

```diff
@@ -2099,71 +2091,74 @@ def get_mimecast_incident_request():
 
 
 def mimecast_incident_api_response_to_markdown(api_response, action_type):
-    incident_code = api_response['data'][0]['code']
-    incident_type = api_response['data'][0]['type']
-    incident_reason = api_response['data'][0]['reason']
-    incident_identified_messages_amount = api_response['data'][0]['Identified']
-    incident_successful_messages_amount = api_response['data'][0]['Successful']
-    incident_failed_messages_amount = api_response['data'][0]['Failed']
-    incident_restored_messages_amount = api_response['data'][0]['Restored']
-    incident_id = api_response['data'][0]['id']
+    incident_code = api_response['data'][0].get('code', '')
```

Comment:
lets initialize a var and use it instead, like `data = api_response.get('data', [{}])[0]`